server/src/engine/queue.ts

```diff
@@ -173,6 +174,8 @@ export async function processQueue () {
       } else if(job.request == jobs.Request.apiAddPkg && !job.src) {
         const base64 = fs.readFileSync(fileURLToPath(job.uri!)).toString('base64')
         socket.sendMessage({...job, base64 })
+      } else if (job.request === jobs.Request.apiAddJar) {
```

Comment:
What is this and why only for add?

Reply:
I think it's actually unneeded since addJar does not have an extra payload.